src/main.cpp

```diff
@@ -642,20 +645,17 @@ bool AcceptToMemoryPoolWorker(CTxMemPool &pool,
     if (pool.exists(hash))
         return state.Invalid(false, REJECT_ALREADY_KNOWN, "txn-already-in-mempool");
 
-    // Check for conflicts with in-memory transactions
+    // Check for conflicts with in-memory transactions and triggers actions at
+    // end of scope (relay tx, sync wallet, etc)
+    respend::RespendDetector respend(pool, *ptx);
+    *isRespend = respend.IsRespend();
+
+    if (respend.IsRespend() && !respend.IsInteresting())
     {
```

Comment:
you already know isRespend, don't need to get it again?

- if (respend.IsRespend() && !respend.IsInteresting())
+if (isRespend && !respend.IsInteresting())

Reply:
Do you prefer to use `*isRespend` over `respend.IsRespend()` for speed or readability (or both)?